enumerables.rb

```diff
@@ -12,13 +12,15 @@ def spicy_foods
 # given an array of spicy foods, **return an array of strings**
 # with the names of each spicy food
 def get_names(spicy_foods)
-  # your code here
+  names = []
+  spicy_foods.each { |food| names << food[:name] }
+  names
```

Comment:
This is exactly what map does. 
```suggestion
  spicy_foods.map { |food| names << food[:name] }
```